Makefile.pre.in

```diff
@@ -918,6 +918,9 @@ oldsharedmods: $(SHAREDMODS) pybuilddir.txt
 		fi; \
 	done
 
+checksharedmods: oldsharedmods sharedmods $(PYTHON_FOR_BUILD_DEPS)
```

Comment:
This target is currently unused. I guess that it will be used later:

>  The script also handles Modules/Setup and will eventually replace setup.py.